.gitignore

```diff
@@ -2,6 +2,9 @@ syntax: glob
 
 ### VisualStudio ###
 
+# Removing toolruntime lock file so that it doesn't get picked up everytime it is generated.
+src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.lock.json
```

Comment:
go ahead and add project.lock.json. We haven't deleted them from this repo yet but we will.